Practical 5/word_occurences_in_string.py

```diff
@@ -0,0 +1,18 @@
+"""
+CP1404/CP5632 Practical
+Program to count the occurrences of words in a string
+"""
+
+text = str(input("Text: "))
+words = text.split()
+words.sort()
+
+counts = dict()
```

Comment:
```suggestion
counts = {}
```
You can also use ```{}``` to create an empty dictionary instead of ```dict()```